packages/hint-compat-api/package.json

```diff
@@ -8,6 +8,8 @@
   },
   "description": "hint to validate if the HTML, CSS, and JavaScript APIs of the project are deprecated or not broadly supported",
   "devDependencies": {
+    "@hint/parser-css": "^2.0.2",
+    "@hint/parser-html": "^2.0.1",
```

Comment:
Do we still need the `parser-html`?

Reply:
We shouldn't need it after the next set of updates because we won't be using the `HTMLEvents` and `HTMLParse` types anymore. Just plain old `element::*` events...

Reply:
Done.